test/e2e/page-objects/cya/checkYourAppeal.js

```diff
@@ -21,8 +21,14 @@ const appellant = testDataEn.appellant;
 function enterDetailsFromStartToNINO(commonContent, language, benefitTypeCode = testDataEn.benefitType.code) {
   const I = this;
 
-  I.enterBenefitTypeAndContinue(commonContent, benefitTypeCode);
-  // I.chooseLanguagePreference(commonContent, 'no');
+  if (language === 'cy') {
+    I.click({ xpath: '/html/body/div[2]/div/p/span/a[2]' });
```

Comment:
Looks like this is failing in Jenkins.

Reply:
Welsh hyper link is disabled on AAT/PR , updated code, now this time it should work

Reply:
I wouldn't access it via the full DOM path, better if you add an ID to the element and access it with that.

Reply:
taken care reviewer comments